ci-operator/step-registry/openstack/provision/sriov-operator/openstack-provision-sriov-operator-commands.sh

```diff
@@ -39,9 +39,6 @@ function wait_for_sriov_pods() {
         done
         if [ -n "${OPERATOR_READY:-}" ] ; then
             echo "sriov-network-operator pods were installed successfully"
```

Comment:
I would put the check here, what do you think?

Reply:
My concern is around workflows that do not use this step "sriov-operator". Example https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/openstack/sriov/openshift-e2e-openstack-sriov-workflow.yaml#L11.
This workflow would never check if the webhook is configured, right?

Reply:
Really good point.
Then I'm fine leaving it here :-)

If you could do the following:
* (optional) move this code to a function for better readability
* call the function ONLY if the webhook is enabled with `oc get sriovoperatorconfig/default -n openshift-sriov-network-operator -o jsonpath='{.spec.enableOperatorWebhook}'`

Reply:
yes. The admission comes at the same time as the pods but the pods have container that aren't Ready right away. So better to check the pods as well. or just the pods. your choice.